libs/libGamedata/lua/CampaignDataLoader.cpp

```diff
@@ -38,14 +40,7 @@ bool CampaignDataLoader::Load()
     }
     return true;
 }
-
 void CampaignDataLoader::Register(kaguya::State& state)
 {
-    state["RTTRCampaignData"].setClass(kaguya::UserdataMetatable<CampaignDataLoader, LuaInterfaceBase>().addFunction(
-      "AddCampaign", &CampaignDataLoader::AddCampaign));
-}
-
-void CampaignDataLoader::AddCampaign(const kaguya::LuaTable& data)
-{
-    CampaignDescription(data, campaignDesc_);
+    state["RTTRCampaignData"].setClass(kaguya::UserdataMetatable<CampaignDataLoader, LuaInterfaceBase>());
```

Comment:
We don't need that Register now, do we?

Reply:
I needed this otherwise it does not work. But i am not a lua expert therefore i am not sure what this line is doing really. In my understanding this is needed to get the connection betweeen lua and our class. Any input is welcome :)